default.nix

```diff
@@ -0,0 +1,9 @@
+{pkgs ? import <nixpkgs> {}}: let
+  lib = pkgs.lib;
+  matchbox = pkgs.buildGoModule {
+  name = "matchbox";
+  src = lib.cleanSource ../matchbox;
```

Comment:
can you explain cleanSource here?

Reply:
This is a recommended macro en development environments: https://gist.github.com/CMCDragonkai/8d91e90c47d810cffe7e65af15a6824c

This filters out some unwanted artifacts (temporary files, git data…).